.github/ISSUE_TEMPLATE/new-feature.yml

```diff
@@ -0,0 +1,55 @@
+name: New Content
+description: For new features or content suggestions
+title: '[NEW]: '
+labels: ['new content']
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to submit this issue. Your input is really appreciated.
+  - type: dropdown
+    id: issue-priority
+    attributes:
+      label: Priority
+      description: Choose one of the options
+      options:
+        - Low
+        - Medium
+        - High
+    validations:
+      required: true
+  - type: textarea
+    id: issue-description
+    attributes:
+      label: Description
+      description: Outline what needs to be changed.
+      placeholder:
+    validations:
+      required: true
+  - type: textarea
+    id: issue-impact
+    attributes:
+      label: How will this impact users?
+      description: Outline how this document change will affect users.
+      placeholder:
+    validations:
+      required: true
+  - type: textarea
+    id: issue-context
+    attributes:
+      label: Context
+      description: Provide any applicable background information.
+      placeholder:
+    validations:
+      required: true
+  - type: textarea
+    id: resolution
+    attributes:
+      label: Suggested resolution
```

Comment:
I feel like this is redundant to the first section that asks you to describe what needs to be changed.

Reply:
I think this one is a bit different. For the "what needs to be changed" , it can be that the version is incorrect or that a feature is deprecated. The suggested resolution section encourages the opener to add a link to the updated content or provide more information (but it is also completely option)

Reply:
But wouldn't a version being incorrect or a feature being deprecated fall under the "Fix content" template instead of the "New content" template? And wouldn't they provide more information in the previous section that says "Provide any applicable background information"?

Reply:
From our meeting: Let's add the same text areas here as in the "fix content" template.